statemachine/callbacks.py

```diff
@@ -1,11 +1,39 @@
+from collections import defaultdict
+from collections import deque
+from typing import Callable
+from typing import Dict
+from typing import List
+
 from .exceptions import AttrNotFound
-from .exceptions import InvalidDefinition
 from .i18n import _
 from .utils import ensure_iterable
 
 
 class CallbackWrapper:
```

Comment:
The `CallbackWrapper` is now only about concrete callback references, and all the metadata attributes are decomposed into a new `CallbackMeta`.